### ISSUE TITLE
Broken "To:"-Field on autoreply + Different mail account name "leaked"

### ISSUE THREAD

**Issue by sephentos** [Mailu/Mailu#2701]
### Environment

- [X] docker compose
- [ ] kubernetes
- [ ] docker swarm

### Version

- Version: `1.9`

## Description

When I enable an autoreply and look at the auto-generated reply, there seems to be something wrong with the email's header. I expect the e-mail address of the actual sender in the "To" field. The automatic reply arrives at the sender, but the "To" field is displayed incorrectly.

What I also find strange is that in the "To:" field a completely different mail address within Mailu is given. This mailbox was neither set up for auto reply, nor was it even the recipient of the actual e-mail. It has no part/context in any part of this scenario.
(No aliases or redirects or something similiar are setup)

Steps to reproduce:

1. Set up an auto reply for A@mailu.tld
2. Send a mail from external@external.tld to A@mailu.tld
3. Mailu replies to external@external.tld with a oddly looking "To:"-Field **which also contains B@mailu.tld which was not a part of everything in this scenario (See screenshot and mail header below)**
4. Open the mail in the mailbox of external@external.tld

![grafik](https://user-images.githubusercontent.com/35430448/225068745-9f4a290c-9d5c-4a30-827a-4661fb16fbcb.png)

```
From: [CORRECT MAILFROM]
To: <SRS1=E426=[MAILU HOSTNAME]==By8E=7G=[EXTERNAL DOMAIN.TLD OF ORIGIN SENDER]=[**DIFFERENT MAIL ACCOUNT IN MAILU**]>
Subject: Urlaub
In-Reply-To: <1df50a6a-c0d4-d35c-ad8d-xxx44e9df04@domain.tld of external sender>
References: <1df50a6a-c0d4-d35c-ad8d-xxx44e9df04@domain.tld of external sender>
Auto-Submitted: auto-replied (vacation)
```

Also replying in e. g. thunderbird to that autoreply results in sending a mail to `SRS1=E426=[MAILU HOSTNAME]==By8E=7G=[EXTERNAL DOMAIN.TLD OF ORIGIN SENDER]=[DIFFERENT MAIL ACCOUNT IN MAILU]` which never could work.

I also find it worth noting that another mailbox name within mailu can be "leaked" through this field.

![grafik](https://user-images.githubusercontent.com/35430448/225072656-9583d30f-a2c6-4793-b1d9-19e7bf3e0e0f.png)

**Comment by nextgens** [comment]
This was fixed in #2640 

There is no plan to backport the fix; Upgrade to master if you need it.

**Comment by sephentos** [comment]
> This was fixed in #2640
> 
> There is no plan to backport the fix; Upgrade to master if you need it.

I have now read a few times that you should upgrade to master. Is there a changelog for the master branch; how else should I know that this and other problems have been fixed (since the website also recommends 1.9 as a stable build I did not thought about switching to master)

Or any ETA for the next release?
